### ISSUE TITLE
Docker install bad css

### ISSUE THREAD

**Issue by CorneelSoete** [onetimesecret/onetimesecret#512]
After installing via docker-compose this is the page that loads at localhost:3000
![Screenshot from 2024-07-29 11-24-42](https://github.com/user-attachments/assets/0e8e99bf-23ca-420d-ae54-76647eac4ad1)

I have the following docker-compose.yml

```
#Onetimesecret
  onetimesecret:
    image: ghcr.io/onetimesecret/onetimesecret:latest
    container_name: onetimesecret
    hostname: onetimesecret
    ports:
      - "3000:3000"
    entrypoint: ["./bin/entrypoint.sh"]
    stdin_open: true
    tty: true
    environment:
      - HOST="localhost:3000"
      - SSL="true"
      - SECRET="CHANGEME"
      - REDIS_URL="redis://CHANGEME@redis:6379/0"
      - COLONEL="corneel.soete@gmail.com"
      - REDIS_PASSWORD="CHANGEME"
#    volumes:
#      - /home/Corneel/Documents/Projects/onetimesecret/Docker/onetimesecret:/app
    depends_on:
      - redis
    networks:
      glpi_network:
        ipv4_address: 172.20.0.4

#Redis
  redis:
    image: redis:bookworm
    container_name: redis
    hostname: redis
    ports:
      - "6379:6379"
    volumes:
      - /home/Corneel/Documents/Projects/onetimesecret/Docker/redis/data:/data
      - /home/Corneel/Documents/Projects/onetimesecret/Docker/redis/redis.conf:/usr/local/etc/redis/redis.conf
    networks:
      glpi_network:
        ipv4_address: 172.20.0.5


# network
networks:
  glpi_network:
    driver: bridge
    ipam:
      config:
        - subnet: 172.20.0.0/24
          gateway: 172.20.0.1
```

**Comment by justin-petermann** [comment]
Hi,
I've the same problem.
Best regards,
Justin.

**Comment by justin-petermann** [comment]
![Capture d’écran 2024-07-30 à 10 50 00](https://github.com/user-attachments/assets/0e95fccc-ea6e-45a9-96de-8c468d022e04)

**Comment by delano** [comment]
There’s a new build pipeline in v0.16 for the UI. I think I know what’s happening here. I’ll confirm and then update the ticket again.

**Comment by delano** [comment]
Yeah it's an easy fix, thankfully. If you add `RACK_ENV=production` to the environment variables and recreate the container, it will load the CSS and other static assets correctly. 

I'll push fixes to the Dockerfile and README as well but this is enough to get you up and running. @justin-petermann Let me know if you need any more details depending on how you're running the container.

This was just released yesterday so thanks for your patience. 

### Examples

#### Docker
```bash
HOST=localhost:3000
SSL=false
COLONEL=admin@example.com
REDIS_URL=redis://host.docker.internal:6379/0
RACK_ENV=production

$ docker run -p 3000:3000 -d --name onetimesecret \
  -e REDIS_URL=$REDIS_URL \
  -e COLONEL=$COLONEL \
  -e HOST=$HOST \
  -e SSL=$SSL \
  -e RACK_ENV=$RACK_ENV \
  onetimesecret/onetimesecret:latest
```

#### Docker compose

```yaml
    environment:
      - HOST="localhost:3000"
      - SSL="true"
      - SECRET="CHANGEME"
      - REDIS_URL="redis://CHANGEME@redis:6379/0"
      - COLONEL="admin@example.com"
      - REDIS_PASSWORD="CHANGEME"
      - RACK_ENV=production 
```

**Comment by justin-petermann** [comment]
Hello !

Thank you, it works!

Best regards,
Justin.

**Comment by delano** [comment]
FYI, these changes are included in the [v0.16.1-RC1](https://github.com/onetimesecret/onetimesecret/releases/tag/v0.16.1-RC1) release and `develop` now. The docker images are building now. 

I'll leave this open for now for visibility and just in case anything else related comes up.
